### PR TITLE
Fix sporadic incorrect logo sizes

### PR DESCRIPTION
**Pull request by natapg**
Closes WEB-62

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ export default function Header() {
             <nav className={styles.navMain}>
               <div className={styles.navLeft}>
                 <Link to='/' className={`${styles.link} ${styles.linkLogo}`} title='Logo'>
-                  <img src='/img/company-logos/fpjs.svg' alt='FingerprintJS' className={styles.logo} />
+                  <LogoSvg alt='FingerprintJS' className={styles.logo} />
```
Comment (tan-90) [inline src/components/Header/index.tsx:51]: Very minor thing, is TS ok with the alt prop? For me it complains when I add `alt` to an imported SVG.

Comment (natapg) [reply src/components/Header/index.tsx:51]: Yes, you are right. I think I should use a title element instead of an alt tag. Thanks

Comment (natapg) [reply src/components/Header/index.tsx]: @tan-90 I added a title element to the SVG file based on this https://css-tricks.com/accessible-svgs/

File: src/img/fpjs.svg [hunk 2]
```diff
@@ -0,0 +1 @@
... 1.48 0 01-1.31.67zm20.93 38.09a1.65 1.65 0 01-1.18-.48 31.66 31.66 0 01-6.72-8.33 26.64 26.64 0 01-3.52-13.69c0-9.37 8.5-17 19-17s18.9 7.6 18.9 17a1.68 1.68 0 01-3.36 0c0-7.65-7-13.87-15.58-13.87s-15.61 6.22-15.61 13.87a23.57 23.57 0 003.11 12.14 29.64 29.64 0 006.2 7.63 1.56 1.56 0 010 2.25 2 2 0 01-1.24.48zm24-5.85a18.86 18.86 0 01-10.38-2.8 16.55 16.55 0 01-8-13.85 1.67 1.67 0 013.34 0 13.39 13.39 0 006.53 11.23 15.37 15.37 0 008.51 2.24 23.94 23.94 0 003.48-.31 1.67 1.67 0 012 1.29 1.59 1.59 0 01-1.38 1.83 25 25 0 01-4 .37zm-6.72 6.51a1.5 1.5 0 01-.44-.07 26.52 26.52 0 01-12.45-6.61 22.44 22.44 0 01-7.28-16.48c0-5.11 4.63-9.29 10.32-9.29s10.31 4.18 10.31 9.29c0 3.36 3.11 6.12 7 6.12s7-2.76 7-6.12c0-11.91-10.89-21.56-24.27-21.56-9.51 0-18.22 5-22.14 12.72a19.37 19.37 0 00-2 8.84 29.57 29.57 0 002.25 11.39 1.53 1.53 0 01-1 2 1.7 1.7 0 01-2.19-.9 33.61 33.61 0 01-2.41-12.49 22.47 22.47 0 012.27-10.23c4.45-8.8 14.33-14.51 25.14-14.51 15.23 0 27.61 11.07 27.61 24.71 0 5.11-4.61 9.27-10.31 9.27s-10.3-4.16-10.3-9.27c0-3.38-3.12-6.12-7-6.12s-7 2.74-7 6.12a19.27 19.27 0 006.34 14.23 23.11 23.11 0 0010.94 5.83 1.57 1.57 0 011.18 1.92 1.64 1.64 0 01-1.57 1.21zm0 0" fill="#f24405"/></svg>
```
Comment (Lisenish) [inline src/img/fpjs.svg:1]: We store local images (the ones which is used only in one place) inside component subfolder, please move it to the `Header` folder

**Review by Lisenish — CHANGES_REQUESTED** [review]

File: src/img/fpjs.svg (hunk 2 above)
Comment (natapg) [reply src/img/fpjs.svg]: done @Lisenish

**Review by Lisenish — APPROVED** [review]

**Pull request merged**